src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public  Random random = new Random();
+    public ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
The fields 'random' and 'colorSupplier' should have access modifiers to encapsulate the internal structure of the class. In most cases, you would want to use 'private' to ensure that these fields are not accessible from outside the class, which is in line with the principle of encapsulation.